File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L3/01/L3.py
import sys

def balik(nomor):
    sisa = 0
    while nomor != 0:
        sisa = nomor % 10
        print(sisa, end='')
        nomor = nomor // 10
    print()

def main():
    print("Enter an integer: ", end='')
    nomor = int(input())
    balik(nomor)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L3/01/L3.kt
package IR

import java.util.*

object L3 {
    @JvmStatic
    fun main(args: Array<String>) {
        var nomor = 0
        print("Enter an integer: ")

        val input = Scanner(System.`in`)

        nomor = input.nextInt()

        balik(nomor)
    }

    fun balik(nomor: Int) {
        var nomor = nomor
        var sisa = 0
        while (nomor != 0) {
            sisa = nomor % 10
            print(sisa)
            nomor = nomor / 10
        }
        println()
    }
}

